model/utils/data_feed.py

```diff
@@ -7,38 +7,24 @@
 import pandas as pd
 import data.mock_data # this is necessary to create mock data if not existent
 
-# pylint: disable=too-few-public-methods
+
 class DataFeed:
     """
     Performs data conversion for generators
     """
-
-    # TODO Build data_feed
-    def __init__(self, data_folder, data=None, length=0):
+    def __init__(self, data_folder):
         self.data_folder = data_folder
-        self.data = data
-        self.length = length
+        self.data = None
+        self.length = None
 
-    def load_historical_data(self, file_name):
+    def load_historical_data(self, data_file):
         """
-        Parser to read prices and turn them into log-returns"""
-        # TODO parser
-
-        if file_name[-3:] == "csv":
-            historical_data = pd.read_csv(Path(self.data_folder, file_name))
-        elif file_name[-3:] == "prq":
-            historical_data = pd.read_parquet(Path(self.data_folder, file_name))
+        reads historical log returns
+        """
+        historical_data = pd.read_parquet(Path(self.data_folder, data_file))
         self.data = np.array(historical_data)
         self.length = len(historical_data)
```

Comment:
Make this part of the initializer and remove this function, it doesn't really make sense at this point.

Reply:
See below, I have the feeling offering some format flexibility would be nice, no?